### DISCUSSION TITLE
GET_LEDGER_DETAIL_VIEW_DATA uses MM/DD/YYYY date format. Shouldn't it use ISO 8601?

### DISCUSSION THREAD

**Discussion by dartheditous** [amzn/selling-partner-api-models#3534] [General]
Since GET_FBA_FULFILLMENT_INVENTORY_RECEIPTS_DATA seems to have been deprecated, I'm migrating to GET_LEDGER_DETAIL_VIEW_DATA.

But it seems that first column in the report, "Date", is using MM/DD/YYYY. This is ambiguous, since it can be hard to distinguish from DD/MM/YYYY.

Shouldn't it be using ISO 8601, as GET_FBA_FULFILLMENT_INVENTORY_RECEIPTS_DATA did?

Is there any way to request ISO 8601 in reportOptions, or a setting in Seller Central?

**Comment by jasonhebert** [comment]
Use the last column (`Date and Time`) in the GET_LEDGER_DETAIL_VIEW_DATA report. This however brings to my attention that the GET_LEDGER_SUMMARY_VIEW_DATA does not have a suitable date column. It contains only the `Date` column with the same issue of format. :(

**Comment by PoByBolek** [comment]
@dartheditous I'm trying to do the same but keep running into `403 Forbidden` errors... Could you tell me which roles you assigned to your app in Seller Central?

Sorry for the off-topic post :-/

**Comment by dartheditous** [comment]
Oh yeah, didn't see that column there, thanks - I was modifying my code for the previous report which took the data from the first column. Does seem a bit of an oversight, though - ISO 8601 should be everywhere!

As for roles, I assume it comes under Amazon Fulfillment. I have everything except Finance and Accounting, Buyer Communication and Buyer Solicitation ticked.

**Comment by nrogers805** [comment]
WHY AS THIS DISCUSSION GONE UNANSWERED BY SP-API REPRESENTATIVE !?
